paper-server/src/main/java/org/bukkit/craftbukkit/block/CraftBlockState.java

```diff
@@ -75,6 +75,7 @@ public void setWorldHandle(LevelAccessor generatorAccess) {
 
     // Returns null if weakWorld is not available and the BlockState is not placed.
     // If this returns a World instead of only a GeneratorAccess, this implies that this BlockState is placed.
+    @org.jspecify.annotations.Nullable
```

Comment:
Don't mix nullable annotations, keep using javax if you don't want to clean things